functions/daemon.go

```diff
@@ -14,6 +14,8 @@ import (
 	"github.com/gravitl/netclient/config"
 	"github.com/gravitl/netclient/local"
 	"github.com/gravitl/netclient/ncutils"
+	nmproxy "github.com/gravitl/netclient/nm-proxy"
```

Comment:
rename the dir to nmproxy

Reply:
why is this change necessary?

Reply:
if the package name matches the directory you eliminate the need for named imports